packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -202,6 +207,25 @@ export class OfficeMockObject {
     this.loaded = false;
   }
 
+  /**
```

Comment:
Delete set?

Reply:
I like having this function internally, as it makes it easier to code in general, as every time you want to add something to the object, you can just call this function.
I could just change the name of this function to "setValue". Would that suffice?